docs/manual/docs/cmd/actions/runscript.md

```diff
@@ -0,0 +1,27 @@
+# runscript action
+
+This action runs an Office 365 CLI command or a set of commands.
```

Comment:
Could you please add `GitHub ` so it is `GitHub action`?

Reply:
Will do when I update the PR